tool/test/org/antlr/v4/test/rt/gen/Generator.java

```diff
@@ -585,6 +585,10 @@ private JUnitTestFile buildParserErrors() throws Exception {
 				"aab",
 				"",
 				"line 1:1 extraneous input 'a' expecting {'b', 'c'}\n");
+		file.addParserTest(input, "SingleTokenDeletionConsumption", "T", "a",
```

Comment:
See the comment I made for `SingleSetInsertionConsumption`; this one relates to the `SingleTokenDeletion` test in the same way.